_includes/compose-eol.md

```diff
@@ -1,6 +1,6 @@
 > **Important**
 >
-> From the end of June 2023 Compose V1 won't be supported anymore and will be removed from all Docker Desktop versions.
+> Compose V1 is no longer supported and will not be available in new releases of Docker Desktop after June 2023.
```

Comment:
Don't we want to merge this now? This implies we have to wait till June to merge

Reply:
This is the winner:
>Compose V1 no longer receives updates and will not be available in new releases of Docker Desktop after June 2023.